WordPress/Classes/Utility/Migrations/PostToPost30To31.m

```diff
@@ -0,0 +1,54 @@
+#import "PostToPost30To31.h"
+#import "AbstractPost.h"
```

Comment:
Make sure to not use any model objects in the migration as they are only valid on the most recent model version. I'd remove this import even though its not used.